### PR TITLE
chore: Remove invalid markdown

### PR DESCRIPTION
**Pull request by casserni**
So pretty silly that this fails the build, but the JSON had an extra `,` making it "invalid" so gatsby couldn't parse it. Also there were some extra important statements from the old docusaurus stuff that would have been an issue as well.

### CODE REVIEW

**Comment by binarylogic** [comment]
Yeah, unfortunately, we need those imports for the current website. Any idea how we can work around that?

File: docs/highlights/2020-10-27-kubernetes-integration.md [hunk 1]
```diff
@@ -34,7 +32,7 @@ import Jump from '@site/src/components/Jump';
     across the board. This [further raises the bar][file_soure_benchmarks] in
     file tailing performance to meet the high demands of large-scale Kubernetes
     environments.
-2.  [**Adaptive Request Currency (ARC)**][adaptive_concurrency_post] -
+3.  [**Adaptive Request Currency (ARC)**][adaptive_concurrency_post] -
```
Comment (lucperkins) [inline docs/highlights/2020-10-27-kubernetes-integration.md:35]: For future reference, Markdown supports auto-inference of list numbers. Like this:

1. Apple
1. Orange
1. Banana

One Weird Trick!

Comment (casserni) [reply docs/highlights/2020-10-27-kubernetes-integration.md:35]: 🤯 🤯 🤯

**Comment by casserni** [comment]
> Yeah, unfortunately, we need those imports for the current website. Any idea how we can work around that?

I think the current website is still serving highlights (and all docs) from the `vector-website` repo since I never did any work to update the old site and to my understanding is how it worked before? So I though the docs and highlights here was only for the new gatsby site?

**Comment by binarylogic** [comment]
Yep, I will remove these components for the time being.

**Pull request merged**